baremetal-coco/dcap/deployment.yaml

```diff
@@ -0,0 +1,159 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: qgs-config
+  namespace: intel-dcap
+data:
+  qgs.conf: |
+    port = 4050
+    number_threads = 4
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sgx-default-qcnl-conf
+  namespace: intel-dcap
+data:
+  sgx_default_qcnl.conf: |
+    {
+      "collateral_service": "https://api.trustedservices.intel.com/sgx/certification/v4/",
+      "pccs_url": https://HOST_IP:8181/sgx/certification/v4",
```

Comment:
Since the DS runs in hostnetwork you should be able to get the HOST_IP as environment variable using downward API. Something like this...
```
  env:
  - name: HOST_IP
         valueFrom:
           fieldRef:
             fieldPath: status.podIP
```

Reply:
My sincere question on this is ... I can get this as a env var, but would I be able to use the env-var directly as part of the conf data that is being defined?

Reply:
I don't think you will. The approach will need to be changed. You can use an initContainer to replace the HOST_IP entry in the config file before the main container start.

Reply:
Yeah, as this is a temporary solution, let's just keep this as it is then, unless you have a really strong opinion on this.

Reply:
How will you replace `HOST_IP` with the actual IP? Sorry I might have missed it..